etc/DependencyInstaller.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Make sure we are on the correct folder before beginning
+cd "$(dirname $(readlink -f $0))/../"
+
+_installORDependencies() {
+    ./tools/OpenROAD/etc/DependencyInstaller.sh -dev
+}
+
+_installCentosCleanUp() {
+    yum clean -y all
+    rm -rf /var/lib/apt/lists/*
+}
+
+_installCentosPackages() {
+    yum update -y
+    yum install -y \
+        libffi-devel \
+        tcl \
+        time \
+        ruby \
+        ruby-devel \
```

Comment:
Which tool depends on ruby?

Reply:
klayout I read in the documentation it needs it as a dependency https://www.klayout.de/build.html